chainer/functions/pooling/roi_average_align_2d.py

```diff
@@ -121,18 +121,20 @@ def _get_bilinear_interp_params(y, x, height, width):
 '''
 
 
-class ROIAlign2D(function.Function):
-
-    """ROI align over a set of 2d planes."""
-
-    def __init__(self, outh, outw, spatial_scale, sampling_ratio=0):
-        for arg in ['outh', 'outw']:
-            value = eval(arg)
-            if not (isinstance(value, int) and value > 0):
-                raise TypeError(
-                    '{} must be positive integer: {}, {}'
-                    .format(arg, type(value), value)
-                )
+class ROIAverageAlign2D(function.Function):
+
+    """ROI average align over a set of 2d planes."""
+
+    def __init__(self, outsize, spatial_scale, sampling_ratio=0):
+        outh, outw = outsize
```

Comment:
How about supporting an integer for `outsize`?

Reply:
ok

Reply:
Updated